service-tests/github.js

```diff
@@ -315,6 +315,20 @@ t.create('Downloads all releases')
     value: Joi.string().regex(/^\w+\s+total$/)
   }));
 
+t.create('downloads for latest release')
+  .get('/downloads/photonstorm/phaser/latest/total.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('downloads'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]?$/)
+  }));
+
+t.create('downloads-pre for latest release')
+  .get('/downloads-pre/photonstorm/phaser/latest/total.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('downloads'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]?$/)
```

Comment:
Hopefully I will merge #1127 in a moment. Could you DRY this up by using `isMetric`?